selfdrive/common/swaglog.cc

```diff
@@ -86,8 +86,12 @@ void cloudlog_e(int levelnum, const char* filename, int lineno, const char* func
 
   if (!s.initialized) s.initialize();
 
+  // if msg is json, parse it
+  std::string err;
+  auto json_msg = json11::Json::parse(msg_buf, err);
```

Comment:
cloudlog_t sends a string from .dump(). this string must be parsed before added to another json string. Otherwise extra backslashes are added.

This was the best solution to the .replace("/", "") situation I found